src/librustc_typeck/diagnostics.rs

```diff
@@ -4677,4 +4677,6 @@ register_diagnostics! {
     E0592, // duplicate definitions with name `{}`
 //  E0613, // Removed (merged with E0609)
     E0627, // yield statement outside of generator literal
+    E0631, // cannot provide explicit type parameters when `impl Trait` is used in
```

Comment:
This error code got taken in the meanwhile, you should pick a different one to avoid an error:
```Rust
[00:03:50] tidy error: duplicate error code: 631
[00:03:50] tidy error: /checkout/src/librustc/diagnostics.rs:2057:     E0631, // type mismatch in closure arguments
[00:03:50] tidy error: /checkout/src/librustc_typeck/diagnostics.rs:4680:     E0631, // cannot provide explicit type parameters when `impl Trait` is used in
[00:03:51] some tidy checks failed
```